azure-sink-connector/src/main/resources/azure-blob-sink-connector-for-apache-kafka.properties

```diff
@@ -0,0 +1,16 @@
+##
+# Copyright 2020 Aiven Oy
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+version=test-version
```

Comment:
This is in main, so it should be packaged for real. I think this needs some sort of placeholder that the build can replace.

Reply:
Did it in the way it's done for other connectors from this repo for now